cluster-autoscaler/core/scale_up.go

```diff
@@ -674,7 +674,7 @@ groupsloop:
 func executeScaleUp(context *context.AutoscalingContext, clusterStateRegistry *clusterstate.ClusterStateRegistry, info nodegroupset.ScaleUpInfo, gpuType string, now time.Time) errors.AutoscalerError {
 	klog.V(0).Infof("Scale-up: setting group %s size to %d", info.Group.Id(), info.NewSize)
 	context.LogRecorder.Eventf(apiv1.EventTypeNormal, "ScaledUpGroup",
-		"Scale-up: setting group %s size to %d", info.Group.Id(), info.NewSize)
+		"Scale-up: setting group %s size to %d instead of %d (max: %d)", info.Group.Id(), info.NewSize, info.CurrentSize, info.MaxSize)
```

Comment:
It would be cleaner if you used something like "setting group % size to %d (previously: %d max %d)". "Instead of" suggests that something altered the decision and info.currentSize was the preferred size. 

@MaciekPytel - do you know if anyone is heavily dependent on the exact stings in events?

Reply:
I am with http://docs.k8s-event-grid-bridge.io/ 😅 

Happy to change it if you want, but it wouldn't really be backwards compatible.

Spoiler alert - Kubernetes needs to adopt CloudEvents with unique event types.